### ISSUE TITLE
Stress is not rendering after update to Star Trek System 1.2

### ISSUE THREAD

**Issue by WesWedding** [WesWedding/sta-enhanced#4]
### Foundry VTT Version

11, 12

### What did you do?

I updated the System to the latest release.

### What did you see happen?

The stress track is missing (but still has a title).

### What did you expect to see?

A stress track!

**Comment by WesWedding** [comment]
The fix introduced in https://github.com/WesWedding/sta-enhanced/commit/a019639fc8dc0b5099de1ab34d713cd92a2bd5c0 needs to be updated in order to make sure it's accessible.

They are not flagged in any way as inputs, nor are they included in tab order.  Someone operating a screen reader would have a hard time finding these new "buttons" and someone using only keyboard can't use them at all.
